src/core/evaluator.js

```diff
@@ -3061,7 +3061,7 @@ var EvaluatorPreprocessor = (function EvaluatorPreprocessorClosure() {
           operation.args = args;
           return true;
         }
-        if (isEOF(obj)) {
+        if (obj === EOF) { // isEOF
```

Comment:
This comment looks a bit superfluous because the line itself is already very clear.